app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java

```diff
@@ -454,8 +464,9 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         searchView = (SearchView) searchItem.getActionView();
         getActivity().setTitle("Eatery");
         AutoCompleteTextView searchTextView =
-                searchView.findViewById(android.support.v7.appcompat.R.id.search_src_text);
+                searchView.findViewById(R.id.search_src_text);
         searchView.setMaxWidth(2000);
+        mFirebaseAnalytics = FirebaseAnalytics.getInstance(getContext());
```

Comment:
is there a reason that this is called below all the other firebase analytics code? shouldn't this be called at the beginning of the file?